sandbox/src/src/index.tsx

```diff
@@ -51,7 +51,14 @@ const cornersArray = ['topLeft', 'topRight', 'bottomLeft', 'bottomRight'] as con
 // const cornersShadowArray = ['topLeftShadow', 'topRightShadow', 'bottomLeftShadow', 'bottomRightShadow'] as const;
 const sidesArray = ['left', 'right', 'top', 'bottom'] as const;
 
-
+/** Generates a sufficiently unique suffix to add to gradient ids and prevent collisions. */
+const generateGradientIdSuffix = (() => {
+  let shadowGradientIdCounter = 0;
+  return () => {
+    shadowGradientIdCounter = (shadowGradientIdCounter + 1) % Number.MAX_SAFE_INTEGER;
+    return new String(shadowGradientIdCounter);
+  };
+})();
```

Comment:
A closure driven counter that loops back to 0 once `Number.MAX_SAFE_INTEGER` is reached. You would need to create 2^53 `<Shadow>` elements before a collision is possible.